pontoon/projects/management/commands/send_review_notifications.py

```diff
@@ -0,0 +1,83 @@
+from collections import defaultdict
+from datetime import timedelta
+from urllib.parse import urlencode
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+from django.urls import reverse
+from django.utils import timezone
+from notifications.signals import notify
+from pontoon.base.models import Translation
+
+
+class Command(BaseCommand):
+    help = "Notify translators about their newly reviewed suggestions"
+
+    def get_description(self, author, notifyData):
+        desc = "Your suggestions have been reviewed:"
+
+        for (locale, project), (approved, rejected) in notifyData.items():
+            url = reverse(
+                "pontoon.translate",
+                kwargs={
+                    "locale": locale.code,
+                    "project": project.slug,
+                    "resource": "all-resources",
+                },
+            )
+            if len(approved) == 1 and len(rejected) == 0:
+                url += "?" + urlencode({"string": approved[0]})
+            elif len(approved) == 0 and len(rejected) == 1:
+                url += "?" + urlencode({"string": rejected[0]})
+            else:
+                url += "?" + urlencode({"author": author.email})
+
+            # Filter out rejections where the author's own suggestion replaced the previous
+            rejected = [x for x in rejected if x not in approved]
+
+            if len(approved) == 0:
+                msg = f"{len(rejected)} Rejected"
+            else:
+                msg = f"{len(approved)} Approved"
+                if len(rejected) > 0:
+                    msg += f", {len(rejected)} Rejected"
+
+            desc += f'\n* {project.name} ({locale.name} · {locale.code}): <a href="{url}">{msg}</a>'
```

Comment:
Let's use HTML lists here, which gives as more styling powers and structure. It might be easier to do with a template, see `suggestion_notification.jinja`.